pyanaconda/modules/network/network_interface.py

```diff
@@ -108,6 +109,15 @@ def IsConnecting(self) -> Bool:
         """
         return self.implementation.is_connecting()
 
+    @property
+    def Capabilities(self) -> List[Int]:
+        """The network backend capabilities
+
+        Supported capabilities:
+        NETWORK_CAPABILITY_TEAM - 1
```

Comment:
Maybe it's just me but it confuses me as `constant -1` expression.

Couldn't we change the comment format similar to:
```
team capability = 1
```

Reply:
Updated